index.html

```diff
@@ -12,9 +12,9 @@
       <button type="button" class="options-button">
         <img
           class="menu-button"
-          src="G:\Julio\Programación\Microverse\Repositories\Julio-Gagliardi-Portfolio-\Enabled.png"
+          src="G:\Julio\Programación\Microverse\Repositories\Julio-Gagliardi-Portfolio\Icons\Enabled.png"
           alt="menu button"
-        >
+        />
       </button>
       <p class="ptitle">Julio's Porfolio</p>
     </header>
```

Comment:
Now that we are done with the linters that have been a nightmare for us. Let's solve some technical issues that do not quite hit the project requirements.

The `hamburger` button should be inside a `nav` element and the `logo text` should be clickable. Therefore, the logo text should be an `<a>` tag. You should have something that looks like this:
![image](https://user-images.githubusercontent.com/38088818/164548984-7a2d4449-710d-47a1-94ea-968b40c68c47.png)